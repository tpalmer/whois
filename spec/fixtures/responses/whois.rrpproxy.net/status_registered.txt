; This data is provided by domaindiscount24.com
; for information purposes, and to assist persons obtaining information
; about or related to domain name registration records.
; domaindiscount24.com does not guarantee its accuracy.
; By submitting a WHOIS query, you agree that you will use this data
; only for lawful purposes and that, under no circumstances, you will
; use this data to
; 1) allow, enable, or otherwise support the transmission of mass
;    unsolicited, commercial advertising or solicitations via E-mail
;    (spam); or
; 2) enable high volume, automated, electronic processes that apply
;    to this WHOIS server.
; These terms may be changed without prior notice.
; By submitting this query, you agree to abide by this policy.

DOMAIN: RRPPROXY.NET

RSP: domaindiscount24.com
URL: register your <a href="http://www.domaindiscount24.com/" rel="dofollow">domain</a> with http://www.domaindiscount24.com

owner-contact: P-ASS1861
owner-organization: Key-Systems GmbH
owner-fname: Alexander
owner-lname: Siffrin
owner-street: Im Oberen Werk 1
owner-city: St. Ingbert
owner-zip: 66386
owner-country: DE
owner-phone: +49 6894 9396850
owner-fax: +49 6894 9396851
owner-email: billing@key-systems.net

admin-contact: P-ASS1861
admin-organization: Key-Systems GmbH
admin-fname: Alexander
admin-lname: Siffrin
admin-street: Im Oberen Werk 1
admin-city: St. Ingbert
admin-zip: 66386
admin-country: DE
admin-phone: +49 6894 9396850
admin-fax: +49 6894 9396851
admin-email: billing@key-systems.net

tech-contact: P-ASS1861
tech-organization: Key-Systems GmbH
tech-fname: Alexander
tech-lname: Siffrin
tech-street: Im Oberen Werk 1
tech-city: St. Ingbert
tech-zip: 66386
tech-country: DE
tech-phone: +49 6894 9396850
tech-fax: +49 6894 9396851
tech-email: billing@key-systems.net

billing-contact: P-ASS1861
billing-organization: Key-Systems GmbH
billing-fname: Alexander
billing-lname: Siffrin
billing-street: Im Oberen Werk 1
billing-city: St. Ingbert
billing-zip: 66386
billing-country: DE
billing-phone: +49 6894 9396850
billing-fax: +49 6894 9396851
billing-email: billing@key-systems.net

nameserver: ns1.p20.dynect.net
nameserver: ns2.p20.dynect.net
nameserver: ns3.p20.dynect.net
nameserver: ns4.p20.dynect.net

; Please register your domains at
; register your <a href="http://www.domaindiscount24.com/" rel="dofollow">domain</a> with http://www.domaindiscount24.com

