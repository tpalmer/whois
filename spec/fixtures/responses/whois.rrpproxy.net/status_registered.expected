#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s raise_error(Whois::PropertyNotSupported)

#updated_on
  should: %s raise_error(Whois::PropertyNotSupported)

#expires_on
  should: %s raise_error(Whois::PropertyNotSupported)


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "domaindiscount24.com"
  should: %s.organization == "Key-Systems GmbH"
  should: %s.url          == "http://www.domaindiscount24.com"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id           == "P-ASS1861"
  should: %s[0].name         == "Alexander Siffrin"
  should: %s[0].organization == "Key-Systems GmbH"
  should: %s[0].address      == "Im Oberen Werk 1"
  should: %s[0].city         == "St. Ingbert"
  should: %s[0].state        == nil
  should: %s[0].zip          == "66386"
  should: %s[0].country_code == "DE"
  should: %s[0].phone        == "+49 6894 9396850"
  should: %s[0].fax          == "+49 6894 9396851"
  should: %s[0].email        == "billing@key-systems.net"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id           == "P-ASS1861"
  should: %s[0].name         == "Alexander Siffrin"
  should: %s[0].organization == "Key-Systems GmbH"
  should: %s[0].address      == "Im Oberen Werk 1"
  should: %s[0].city         == "St. Ingbert"
  should: %s[0].state        == nil
  should: %s[0].zip          == "66386"
  should: %s[0].country_code == "DE"
  should: %s[0].phone        == "+49 6894 9396850"
  should: %s[0].fax          == "+49 6894 9396851"
  should: %s[0].email        == "billing@key-systems.net"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id           == "P-ASS1861"
  should: %s[0].name         == "Alexander Siffrin"
  should: %s[0].organization == "Key-Systems GmbH"
  should: %s[0].address      == "Im Oberen Werk 1"
  should: %s[0].city         == "St. Ingbert"
  should: %s[0].state        == nil
  should: %s[0].zip          == "66386"
  should: %s[0].country_code == "DE"
  should: %s[0].phone        == "+49 6894 9396850"
  should: %s[0].fax          == "+49 6894 9396851"
  should: %s[0].email        == "billing@key-systems.net"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(4)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.p20.dynect.net"
  should: %s[0].ipv4 == nil
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.p20.dynect.net"
  should: %s[1].ipv4 == nil
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns3.p20.dynect.net"
  should: %s[2].ipv4 == nil
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns4.p20.dynect.net"
  should: %s[3].ipv4 == nil
